crates/astria-cli/src/sequencer/block_height.rs

```diff
@@ -0,0 +1,63 @@
+use astria_sequencer_client::{
+    Client as _,
+    HttpClient,
+};
+use clap::Subcommand;
+use color_eyre::eyre::{
+    self,
+    WrapErr as _,
+};
+
+#[derive(Debug, clap::Args)]
+pub(super) struct Command {
+    #[command(subcommand)]
+    command: SubCommand,
+}
+
+impl Command {
+    pub(super) async fn run(self) -> eyre::Result<()> {
+        let SubCommand::Get(get) = self.command;
+        get.run().await
+    }
+}
+
+#[derive(Debug, Subcommand)]
+enum SubCommand {
+    /// Get the current block height of the Sequencer node
+    Get(Get),
+}
+
+#[derive(clap::Args, Debug)]
+struct Get {
+    /// The url of the Sequencer node
+    #[arg(
+        long,
+        env = "SEQUENCER_URL",
+        default_value = crate::DEFAULT_SEQUENCER_RPC
+    )]
+    sequencer_url: String,
+    /// The chain id of the sequencing chain being used
+    #[arg(
+        long = "sequencer.chain-id",
+        env = "ROLLUP_SEQUENCER_CHAIN_ID",
+        default_value = crate::DEFAULT_SEQUENCER_CHAIN_ID
+    )]
+    sequencer_chain_id: String,
```

Comment:
I don't think this is used here?

Reply:
No, it's not. But fixing this is out of scope of this PR. This bit is copied verbatim from the previous version.